secrets/resource_secret_acl.go

```diff
@@ -33,23 +37,66 @@ func ResourceSecretACL() common.Resource {
 	}
 	return common.Resource{
 		Schema: s,
-		CanSkipReadAfterCreateAndUpdate: func(_ *schema.ResourceData) bool {
-			return true
-		},
 		Create: func(ctx context.Context, d *schema.ResourceData, c *common.DatabricksClient) error {
 			w, err := c.WorkspaceClient()
 			if err != nil {
 				return err
 			}
 			var req workspace.PutAcl
 			common.DataToStructPointer(d, s, &req)
-			err = w.Secrets.PutAcl(ctx, req)
-			if err != nil {
-				return err
+			
+			// Retry logic: up to 3 attempts with 90-second intervals
+			const maxRetries = 3
+			retryInterval := 90 * time.Second
+			
+			// Allow tests to override retry interval
+			if testInterval := os.Getenv("DATABRICKS_SECRET_ACL_TEST_RETRY_INTERVAL_MS"); testInterval != "" {
+				if ms, err := strconv.Atoi(testInterval); err == nil {
+					retryInterval = time.Duration(ms) * time.Millisecond
+				}
+			}
+			
+			for attempt := 1; attempt <= maxRetries; attempt++ {
```

Comment:
we can use `retry` package for it: https://github.com/databricks/terraform-provider-databricks/blob/main/vectorsearch/resource_vector_search_index.go#L22

Reply:
will do thanks